index.html

```diff
@@ -21,11 +23,12 @@
           </section>
      </nav>
   
-
   <div class="container">
     <section class="text1">
-      <h1>I'm Suleiman Yusuf Gacheru.
-        <p>Glad to see you! </p></h1>
+
+      <h1>I'm Suleiman Yusuf Gacheru. Glad to see
+         you! </h1>
+      
     </section>
     <section class="intro">
```

Comment:
- [ ] Nice work wrapping the paragraph with a `p` tag :+1: However it would be nice to remove this `<section class="intro">` tag from your code. You can add the `class="intro` to the `p` tag, that will make your code clean and DRY :muscle: 

The `section` tag should use only for section in your page. [Read more about `section` tag](https://www.w3schools.com/tags/tag_section.asp)